### PR TITLE
Add desktop style

### PR DESCRIPTION
**Pull request by nmeziany**
In this part of the project I:
- Create new CSS file for desktop style of the project.
- Fix the issues related to those changes

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -1,360 +1,373 @@
...
-}
-
-hr {
-  border: 3px solid #0d151e;
-  width: 134px;
-  border-radius: 10px;
-  background-color: #0d151e;
+@import url("desktop-style.css") screen and (min-device-width: 768px);
+
+@media (max-width: 375px) {
```
Comment (sja-thedude) [inline style.css:3]: - [x] Kindly remove @media max-width as the width more than this is causing the problem.
- As seen below:
Screenshot 2022-03-26 at 11 57 56 PM

File: index.html [hunk 2]
```diff
@@ -63,139 +68,152 @@
...
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul>
+              <li class="lang">html</li>
+              <li class="lang">css</li>
+              <li class="lang">javaScript</li>
+            </ul>
+            <a class="moreabout" href="#">See Project</a>
+        </div>
```
Comment (sja-thedude) [inline index.html:158]: - [ ] Kindly follow the Figma guidelines, design and fix the cards. As you can see all the cards in even numbers are flex reverse where Multi post stories are placed first and the image second. Fix it.
- Your design:
Screenshot 2022-03-27 at 12 07 28 AM
Screenshot 2022-03-27 at 12 07 33 AM

Figma design:
Screenshot 2022-03-27 at 12 07 03 AM

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @nmeziany, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Good PR Description 
✔️ Great on working on the project so far
✔️ Professional Readme

## Required 
## MOBILE VERSION:
- [x] Kindly make sure to remove extra margin (left and right side) from the home page section, about myself section and the contact form section.
- [x] Kindly fix the contact form as per Figma design following the margin, spacing.
- [x] Kindly fix all the cards to align them in the centre as you have noticed that it has more margin on one side.
- [x] Kindly fix the arrow icons in the about myself section as per the Figma design.
Your design: 
Screenshot 2022-03-27 at 12 04 23 AM
Screenshot 2022-03-27 at 12 04 51 AM
Screenshot 2022-03-27 at 12 04 57 AM

Figma design:
Screenshot 2022-03-27 at 12 01 54 AM
Screenshot 2022-03-27 at 12 02 07 AM

- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 1 above)
Comment (nmeziany) [reply style.css:3]: Changes done!
It is working after the modification

**Pull request closed without merging**